test/jdk/java/lang/Thread/virtual/Starvation.java

```diff
@@ -25,7 +25,7 @@
  * @requires vm.continuations
  * @library /test/lib
  * @bug 8345294
- * @run main/othervm --enable-native-access=ALL-UNNAMED Starvation 100000
+ * @run main/othervm/timeout=200/native --enable-native-access=ALL-UNNAMED Starvation 100000
```

Comment:
Did you mean to add /timeout=200 to this test?

Reply:
Test test/jdk/java/lang/Thread/virtual/Starvation.java execute over 120 seconds on my local environment. So I add timeout=200 also.